libdnf/dnf-context.cpp

```diff
@@ -983,6 +998,21 @@ dnf_context_set_keep_cache(DnfContext *context, gboolean keep_cache)
     priv->keep_cache = keep_cache;
 }
 
+/**
+ * dnf_context_set_enable_filelists:
+ * @context: a #DnfContext instance.
+ * @enable_filelists: %TRUE to download and parse filelist metadata
+ *
+ * Enables or disables download and parsing of filelists.
+ **/
+void
+dnf_context_set_enable_filelists (DnfContext     *context,
+                                  gboolean        enable_filelists)
```

Comment:
Hmm, actually, maybe we should have a new `setup_sack_with_flags` which defaults to `NONE`? E.g. for auto-updates, I'd like to be able to specify `UPDATEINFO`. And also to add a new flag to not fall back to updating an out-of-date repo.